library/ecdsa.c

```diff
@@ -64,7 +64,7 @@ static int derive_mpi( const mbedtls_ecp_group *grp, mbedtls_mpi *x,
 cleanup:
     return( ret );
 }
-
+#if !defined (MBEDTLS_ECC_ALT)
```

Comment:
Since it is only alternative for functions, and not for the whole module, it should be an alternative function implementation, and not alternative module implementation